database.py

```diff
@@ -1,9 +1,7 @@
 from pprint import pprint
```

Comment:
I think we can remove `pprint` import as well since it is not being used anywhere in the code.

Reply:
@nooneofyourbuisness Could you please remove this line ?
```python
from pprint import pprint
```

Reply:
Just remove this line and I think we are good to merge.

Reply:
done removed `from pprint import pprint` also added try except catch on line 169-172 which i forgot previously